akka-distributed-cluster-docs/src/main/paradox/feature-summary.md

```diff
@@ -15,7 +15,7 @@ With Projections over gRPC you could have the same advantages of loosely coupled
 without the burden and cost of operating the broker infrastructure. As a bonus, you would gain low-latency cross
 region delivery of messages.
 
-![projection-over-grpc.png](images/projection-over-grpc.svg)
+![Diagram showing services using projections over gRPC with one producer and two consumers](images/projection-over-grpc.svg)
```

Comment:
I just realised the first part is what is put in the `alt` text attribute for the images, which is there for visually impaired/screen readers etc, so better put something useful than the filename.